tutorials/textcat_goemotions/project.yml

```diff
@@ -8,6 +8,8 @@ vars:
   gpu_id: -1
   # Change this to "bert" to use the transformer-based model
   config: "cnn"
+  # Training Directory
+  training_directory: "training"
```

Comment:
I'm not sure we want to put this behind a variable. If we do though, we need to update many more references in this .yml file: the `directories` at L16, the `outputs` and `deps` of some of the commands, and some of the `script` instructions. I actually fear that it'll become a bit unreadable. On the other hand, if we just hardcode it as "training", it's not much work to rename the directory afterwards... As these are example projects, there's a trade-off between clarity of the code and generalizability...

Reply:
Ok, I can hard code it in line 77 and drop the variable.

Reply:
Updated